formats/MrssFormat.php

```diff
@@ -52,6 +53,7 @@ public function stringify(){
 		$feed->setAttribute('version', '2.0');
 		$feed->setAttributeNS('http://www.w3.org/2000/xmlns/', 'xmlns:atom', self::ATOM_NS);
 		$feed->setAttributeNS('http://www.w3.org/2000/xmlns/', 'xmlns:media', self::MRSS_NS);
+		$feed->setAttributeNS('http://www.w3.org/2000/xmlns/', 'xmlns:itunes', self::ITUNES_NS);
```

Comment:
Hmm, maybe we can even move this line inside the `if ($itemDuration !== null)` block. It is idempotent so it won’t matter that we all it multiple times.